Create style-guide.md (#184)
w3c/wai-website-theme: _docs/style-guide/tone-language-words.md

<b>Voice recognition</b> identifies who is speaking by analyzing their unique vocal characteristics.

> [!TIP]
> Example:
>
> Voice recognition is used for security features like [voice biometrics](https://www.google.com/search?cs=0&sca_esv=5c87a0e2847a9f64&q=voice+biometrics&sa=X&ved=2ahUKEwig3JPq5P2PAxXBX0EAHaN5L50QxccNegQIBBAD&mstk=AUtExfCTFIZsHUsJYQlPctWxIoA2kmQg8uDu_ubh8nWfaA6wg2THu_veJsrxggz6YXSIV_eNO7eUefMTbWORk5rTmaCfpJ_ZWYk12YYupkST3WNfB4ieGRoRwG_uXMvRECVCGFdm82NC0snmLjNkP00ZVM8soQM_T00otMJ-m1D3GNUiQGc&csui=3) to authenticate a user.

#### URL (Uniform Resource Locator)

A string of characters that identifies a resource. It provides information on how to locate and access the resource. Use this term when referring to links to webpages.

> [!TIP]
> Example:
>

> > https://www.w3.org/WAI/index.html — this identifies the resource, the WAI homepage, as a webpage at the w3.org domain accessed using the secure HTTP protocol.

Reviewer: ```suggestion
> https://www.w3.org/WAI/ — this identifies the resource, the WAI homepage, as a webpage at the w3.org domain accessed using the secure HTTP protocol.
```

Strong preference for the canonical URL.

* “deaf” (with a lower case letter): A medical term to describe anyone who does not hear very much or at all. 

 **Note:** A person who is hard of hearing may not use sign language or identify with Deaf culture)

See also [Terms to avoid relating to disability / disabled people](#terms-to-avoid-relating-to-disability-disabled-people).

### Inclusive language

Use gender-neutral language, as feasible.

<aside class="example">
<i>Example:</i><br>

Use “chair”, not “chairman”.
</aside>

Avoid assumptions about gender.

<aside class="example">
<i>Example:</i><br>

Not all builders are male.
</aside>

See also:

* [Disability language](/style-guide/content-wording/#disability-language)
* [Pronouns: Gender inclusivity and translation considerations](/style-guide/content-wording/#pronouns-gender-inclusivity-and-translation-considerations)

#### Pronouns: Gender inclusivity and translation considerations

When writing for a global audience, use pronouns that are both **inclusive** and **easy to translate**. 

##### Use a plural subject to avoid (he/she, his/her)

> Avoid assuming gender by skipping 'he/she and his/her', where possible. Use a <b>plural subject</b> instead.

Reviewer: ```suggestion
Avoid assuming gender by skipping 'he/she' and 'his/her', where possible. Use a <b>plural subject</b> instead.
```

* <b>use</b> the content to complete tasks

#### Techniques for writing in plain language

##### Abbreviations

Provide the full term with the abbreviation on first use — there are [a few ways you can do this](https://www.w3.org/WAI/WCAG22/Techniques/general/G102#related).

**Exceptions:** See [Abbreviations — Style for different content types](/style-guide/style-different-content-types/#abbreviations).

##### Contractions

Use positive contractions and avoid negative contractions.

<aside class="example">
<i>Examples:</i><br>

<ul>
  <li>Avoid: “cannot” instead of “can’t”; “do not” instead of don’t, and “will not” instead of “won’t”</li>
  <li>Use: it’s, that’s, you’re, they’ll, we’ve, let’s</li>
</ul>
</aside>

##### Personal pronouns

Use “you” and “your” when talking to the reader; and “we” when talking on behalf of W3C. 

##### Structure

Break up chunks of information to make it easier to parse using the guidance under [Structure](/style-guide/structure-presentation/#structure).

##### Verbs

Use a verb instead of a noun describing an action.

<aside class="example">
<i>Examples:</i><br>

Example 1:

<ul>
  <li>Avoid: We conducted an analysis of the results.</li>

>   <li>Use: We analyzed the results.</li>

Reviewer: That's probably a good example of a use of "we" that, in many cases, cannot be "on behalf of W3C".

<b>Speech recognition</b> converts what is said into text for speech-to-text (STT) transcription, virtual assistants, and other speech user interfaces.

<aside class="example">
<i>Example:</i><br>

Speech recognition powers virtual assistants to understand commands.
</aside> 

<b>Voice recognition</b> identifies who is speaking by analyzing their unique vocal characteristics.

<aside class="example">
<i>Example:</i><br>

Voice recognition is used for security features like voice biometrics to authenticate a user.
</aside> 

#### URL (Uniform Resource Locator)

A string of characters that identifies a resource. It provides information on how to locate and access the resource. Use this term when referring to links to webpages.

<aside class="example">
<i>Example:</i><br>

https://www.w3.org/WAI/ — this identifies the resource, the WAI homepage, as a webpage at the w3.org domain accessed using the secure HTTP protocol.
</aside> 

#### URI (Uniform Resource Identifier) 

A string of characters that identifies a resource. It does not always describe how to locate the resource. Use this term when referring to a resource identifier.

<aside class="example">
<i>Example:</i><br>

> mailto:kevin@w3.org — this identifies an email resource.

Reviewer: ```suggestion
mailto:accessibility@example.org — this identifies an email resource.
```

Unless @iadawn would like to get more spam 😅
